pages/en/participate/reporting.mdx

```diff
@@ -7,20 +7,20 @@ export default Page({
 });
 
 # Reporting Issues, Bugs, & Feature Requests 
-Learn how to report issues and participate in Mina’s Bug Bounty Program
```

Comment:
@jasongitmail If we add a space between the title (#) and the next line, the formatting will change.

Reply:
Ah will push a new commit for that. My editor adds it with markdown prettier